blocks/test/fixtures/core__author.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "uid": "_uid_0",
+        "name": "core/author",
+        "isValid": true,
+        "attributes": {
+			"hideBio": false,
```

Comment:
Wonder why tabbing is off here, since this is generated. 🤷‍♂️

Reply:
Here in github it looks like it has wrong tabbing but I checked in the editor the tabs were right, looks like this is related with github visualization.

Reply:
It's a difference of tabs and spaces. Depending on how your editor is configured, it might be showing tabs as 4 spaces, appearing aligned.